### DISCUSSION TITLE
ao <- available_outcomes() Error: $ operator is invalid for atomic vectors

### DISCUSSION THREAD

**Discussion by lhs123628** [MRCIEU/TwoSampleMR#318] [Q&A]
I‘m using version 0.5.6 and R has been upgrate to 4.1.1. And when I tried to run **_available_outcomes()_**, it threw the following error message.
`> library(TwoSampleMR)`
`> ao <- available_outcomes()`
`API: public: http://gwas-api.mrcieu.ac.uk/`
`Error: $ operator is invalid for atomic vectors`

Any idea to solve?
Thanks!

**Comment by alicechen14** [comment]
have you fixed this? they said it's because of bad connection to the internet.

**Comment by lhs123628** [comment]
we didn't encount this error, maybe you can look for help in github.



---- Replied Message ----
| From | ***@***.***> |
| Date | 04/14/2024 13:37 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [MRCIEU/TwoSampleMR] ao <- available_outcomes() Error: $ operator is invalid for atomic vectors (Discussion #318) |

have you fixed this? they said it's because of bad connection to the internet.
